src/App.ts

```diff
@@ -297,6 +297,16 @@ export default class App {
       this.logLevel = logLevel ?? LogLevel.DEBUG;
       // Set SocketMode to true if one wasn't passed in
       this.socketMode = socketMode ?? true;
+
+      axios.get('https://api.github.com/repos/slackapi/bolt-js/releases/latest').then((res) => {
+        // res.data.tag_name is "@slack/bolt@<version>"
+        const version = res.data.tag_name.split('@')[2];
```

Comment:
We would like to extract these data extraction to a function and write some tests for it.